scarlet/renderer.py

```diff
@@ -185,7 +185,6 @@ def __init__(self, data_frame, model_frame, convolution_type="fft", padding=10):
         ll = np.round(pixel_in_model_frame.min(axis=0)).astype("int")
         ur = np.round(pixel_in_model_frame.max(axis=0)).astype("int") + 1
         bounds = (ll[0], ur[0]), (ll[1], ur[1])
-        data_box = model_frame.bbox[0] @ Box.from_bounds(*bounds)
```

Comment:
why is this removed? It used to set `self.slices`, which is apparently already gone, but it was needed below...